Gemfile

```diff
@@ -26,3 +26,4 @@ end
 
 gem 'rubocop', require: false
 
+gem 'graphql_upgrader', path: '/Users/willian/src/graphql_upgrader'
```

Comment:
👋

Reply:
Shouldn't this point to github?

Reply:
@esjee you're right, I'm using the local gem to develop the upgrader and use this PR to show the world what it can do. I'll remove the /Gemfile(?:.lock)?/ changes when I'm done.